### ISSUE TITLE
Quotes in ffmpeg parameter

### ISSUE THREAD

**Issue by Kedryn** [HaveAGitGat/Tdarr#791]
**Describe the bug**
Writing a plugin, i need to specify a 
-vf scale=-1:'min(720,ih)'
in the fffmpeg parameters
But i can't. Tdarr keeps stripping single quotes from the line, and there is no other combination of double-single quotes that works

the result is:

[Parsed_scale_0 @ 00000226a180c2c0] [Eval @ 0000009a67ffdf80] Missing ')' or too many args in 'min(720'
[Parsed_scale_0 @ 00000226a180c2c0] Cannot parse expression for height: 'min(720'
[AVFilterGraph @ 00000226a778ca40] Error initializing filter 'scale' with args '-1:min(720'

**To Reproduce**
simply put that parameter in a plugin
Instead, it works if i use it in  "basic video" 

**Expected behavior**
Expect Tdarr to not strip quotes

**Comment by HaveAGitGat** [comment]
Do you have the job report? https://docs.tdarr.io/docs/other/job-reports/

I just tried running this with `Tdarr_Plugin_075a_Transcode_Customisable` using args `<io> -c:v libx265 -vf scale=-1:'min(720,ih)'` and it worked fine for me:

```
2023-06-01T06:33:54.741Z yi5UF-Rxd:Node[node2]:Worker[vast-vole]:Create transcode args
2023-06-01T06:33:54.743Z yi5UF-Rxd:Node[node2]:Worker[vast-vole]:Args:  -i "C:/Transcode/Source Folder/sample__1080__libx264__aac__30s__video.mp4" -c:v libx265 -vf scale=-1:'min(720,ih)' "C:/Transcode/Cache Folder/sample__1080__libx264__aac__30s__video-TdarrCacheFile-f29grzktd.mkv"
```

![image](https://github.com/HaveAGitGat/Tdarr/assets/43864057/e7178f2e-27cd-47c8-bc37-af54b13451f0)

**Comment by Kedryn** [comment]
I assume that "customizable args" means a textbox where you write that
parameter.

Try, instead, injecting that in the code

Right now i'm not at home.

Il gio 1 giu 2023, 07:38 HaveAGitGat ***@***.***> ha scritto:

**Comment by HaveAGitGat** [comment]
Yeah that's right, like this:
![image](https://github.com/HaveAGitGat/Tdarr/assets/43864057/3ad9d5d6-1696-4004-8cc3-1d5383357aa8)

**Comment by HaveAGitGat** [comment]
Reopen if needed thanks.
